### ISSUE TITLE
Screen reader settings are broken

### ISSUE THREAD

**Issue by chrstnb** [google-gemini/gemini-cli#7777]
Currently, setting 
ui: { accessibility: { screenReader: true }} does not work

**Comment by hugomurillomtz** [comment]
@chrstnb this is ready for review if you may please: https://github.com/google-gemini/gemini-cli/pull/8730
